esphome/components/xiaomi_mccgq02hl/binary_sensor.py

```diff
@@ -0,0 +1,74 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, binary_sensor, esp32_ble_tracker
+from esphome.const import (
+    CONF_MAC_ADDRESS,
+    CONF_ID,
+    CONF_BINDKEY,
+    CONF_OPEN,
+    CONF_DEVICE_CLASS,
+    CONF_HAS_LIGHT,
+    CONF_BATTERY_LEVEL,
+    UNIT_PERCENT,
+    ICON_BATTERY,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_LIGHT,
```

Comment:
It fails to compile if not at least the device class opening is imported. The others for consistency with other Xiaomi sensors sending a battery status.

```suggestion
    DEVICE_CLASS_LIGHT,
    DEVICE_CLASS_OPENING,
    STATE_CLASS_MEASUREMENT,
    ENTITY_CATEGORY_DIAGNOSTIC
```